source/elements/oneDPL/source/parallel_api/iterators.rst

```diff
@@ -429,9 +429,13 @@ of the iterator class.
     {
         It1 first;
         It2 second;
-        friend auto is_onedpl_indirectly_device_accessible(it_pair) ->
+        friend auto
+        is_onedpl_indirectly_device_accessible(it_pair) ->
             std::conjunction<oneapi::dpl::is_indirectly_device_accessible<It1>,
-                             oneapi::dpl::is_indirectly_device_accessible<It2>>;
+                             oneapi::dpl::is_indirectly_device_accessible<It2>>
+        {
```

Comment:
Does this need to be added to the specification for the implementation in https://github.com/uxlfoundation/oneDPL/pull/2241 to remain compliant?  To me the introduction of the body definition in oneDPL is an implementation detail that's being done to address compiler warnings.  Another implementation that didn't provide the definition would still have the same semantic behavior, correct?

Reply:
>Another implementation that didn't provide the definition would still have the same semantic behavior, correct?

Yes, this is how I understand it. We are not changing semantics, just avoiding an example or implementation suggestion for users which may get them to encounter warnings, or the perception of an error. No language needs to change here, its still valid to use a hidden friend, and its still valid to omit the body of the function.  If you do them both at the same time, then you open yourself up to this (false) warning.

Reply:
Ok. As this is being provided as an example it looks good to me.